nixos/modules/services/misc/nix-gc.nix

```diff
@@ -21,13 +21,45 @@ in
       };
 
       dates = mkOption {
-        default = "03:15";
         type = types.str;
+        default = "weekly";
+        example = "03:15";
```

Comment:
Same as PR #107957, I don't think you should change the current default.

Reply:
Or if you do change it, please document it in https://github.com/NixOS/nixpkgs/blob/7b968c79a15796c524b75a0e43d643e36aa10e97/nixos/doc/manual/release-notes/rl-2103.xml#L86.

Reply:
I was thinking about this. Is it wise / OK / expected to run garbage collector every day?

Reply:
I basically run at every update, so considering that the default to updates is to run at every `3:30` I think it is fine (and I also think this is why the default is `4:40`, so it runs after the updates finishes).

But I am also ok to changing the default, my only problem is changing the default and not documenting it, since someone somewhere maybe depending on the old default.